deploy/install.sh

```diff
@@ -51,42 +118,124 @@ if [ -z "${NAMESPACE}" ]; then
     exit 1
 fi
 
-if kubectl -n "${NAMESPACE}" get serviceaccount/"${SERVICEACCOUNT}" &> /dev/null; then
+kubectlApplySecret () {
+    local secretName="$1"
+    local secretTemplate="$2"
+    local username="$3"
+    local password="$4"
+    # shellcheck disable=SC2002
+    cat "${secretTemplate}" | sed "s/{{name}}/${secretName}/" | sed "s/{{username}}/${username}/" | sed "s/{{password}}/${password}/" | "${KUBECTL_BIN}" -n "${NAMESPACE}" apply -f -
+}
+
+installSecret () {
```

Comment:
It is good that one manage these secrets entirely outside so that nothing happens when the secrets  already exist. To me the function name conveys that it always installs. Perhaps `ensureSecret` or `ensureOrInstallSecret ` would be better names?